src/scripts/components/component.es6

```diff
@@ -96,7 +96,7 @@ module.exports = class Component {
 	destroy() {
 		this.detachListeners()
 
-		for (let prop in this) {
+		for (const prop in this) {
```

Comment:
Na `const` v konstrukci `for` pozor, v tomto případě fungovat bude, ale při použití `for ( ... ; ... ; ... )` nebude, protože tam se přiřazení vykoná před smyčkou.
Doporučoval bych tedy pro konzistenci používat ve smyčkách `let`.